README.md

```diff
@@ -1,6 +1,7 @@
 # ImmutableRecord
 
-TODO: Write a gem description
+ImmutableRecord provides simple immutable data structures with accessor
+methods and ActiveRecord style mass-assignment initialization.
```

Comment:
If you want people to use the gem, you might want to avoid quoting ActiveRecord as an inspiration :p